code/modules/reagents/reagent_containers/cups/glassbottle.dm

```diff
@@ -579,6 +579,9 @@
 			isGlass = FALSE
 	return
 
+/obj/item/reagent_containers/cup/glass/bottle/molotov/throw_impact(atom/hit_atom, datum/thrownthing/throwingdatum)
```

Comment:
```suggestion
/obj/item/reagent_containers/cup/glass/bottle/molotov/throw_impact(atom/hit_atom, datum/thrownthing/throwingdatum, do_splash = FALSE)
```
Just so they're consistent

Reply:
Aye

Reply:
Wait do we in this case even need to specify that we call ..() with do_splash =  FALSE? Or is that optional in DM?